### DISCUSSION TITLE
Improve order of fields in Specify Network table

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5799] [Feature Requests]
@acbentley:

> I think there is possibly a more logical ordering of these fields that would make more sense to users - Catalog Number, Collection Code, Basis of Record, Dataset Name, Dataset key, GBIF Publisher ID, Institution Code, Family, Genus, Species, Scientific Name, Accepted Scientific Name, Field Number, Date (can we add full date field?), Year, Month, Day, Country, State, County, Locality, Latitude, Longitude, Datum, Preparations, Modified by, Record ID (collapse GBIF, iDigBio and Specify ID's into one row, Modified Date

> I note that collectors is not included.  Can we remove Recorded By (of little use) and replace with Collected by and insert in order above just after date fields?

**Comment by maxpatiiuk** [comment]
From @beach53:

> This is a snippet of the SN Collection Object popup dialog window from the CO form.  Is "DataSet Key" "GBIF Dataset Key"?  If yes can we add GBIF to the front of that?  
[@Aimee](https://specifydev.slack.com/team/UQRSQHP60)
 While we are mucking around in there, "Accepted Scientific Name" near the bottom in the GBIF column, wouldn't that make more sense to be near the other names at the top?  Maybe in the second row?
One last question, the "Specify Record ID" or "Specify GUID" is absent.  GBIF should have that, I know we have talked about this.  Is the issue that it is in an awkward place in GBIF and we cannot retrieve it without difficulty?
> 
![image](https://github.com/specify/specify7/assets/40512816/137058a8-bdff-4bf2-81a4-28dd2aa707e0)

**Comment by beach53** [comment]
Thanks Max
